<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMainWin</class>
 <widget class="QMainWindow" name="QtMainWin">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenStDepth</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuLandmarks">
    <property name="title">
     <string>Landmarks</string>
    </property>
    <addaction name="actionLoad_Land"/>
    <addaction name="actionSaveLandmarks"/>
    <addaction name="actionClearMarks"/>
    <addaction name="actionResetMarks"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLandmarks"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="toolTip">
    <string>Status bar</string>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionOpen_Dir">
   <property name="text">
    <string>Open Directory</string>
   </property>
  </action>
  <action name="actionSaveLandmarks">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionView_in_3D_Land">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View in 3D</string>
   </property>
  </action>
  <action name="actionLoad_Land">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionConfigCamera">
   <property name="text">
    <string>Parameters</string>
   </property>
  </action>
  <action name="actionClearMarks">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionResetMarks">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset Viewer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>QtMainWin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
